gui/src/GUIUpdater/GUIUpdater.cpp

```diff
@@ -411,7 +411,6 @@ void Gui::GUIUpdater::updatePlayerRessourceCollecting(const std::vector<std::str
         for (auto &player : team.getPlayers()) {
             if (player.getId() == args[0]) {
                 player.setState(Gui::Player::PlayerState::COLLECT);
-                _network.get()->sendMessageServer("pin " + std::to_string(player.getId()) + "\n");
                 _gameData.get()->getTile(player.getPosition().first, player.getPosition().second).inventory.removeResource(args[1], 1);
```

Comment:
```suggestion
                player.inventory.addResource(args[1], 1);
                _gameData.get()->getTile(player.getPosition().first, player.getPosition().second).inventory.removeResource(args[1], 1);
```
Can you add this line please, and the same but with **removeResource** in updatePlayerRessourceDropping l.365